.circleci/config.yml

```diff
@@ -284,9 +284,9 @@ references:
   docker-images:
     # These versions should try to match what we run in production.
     - &python "cimg/python:3.9"
-    - &redis "redis:2.8"
+    - &redis "redis:6.2"
```

Comment:
that's quite an upgrade!

Reply:
Note that it was already done in production and in our local images (https://github.com/mozilla/addons-server/pull/19001) - this is just for consistency. Ultimately we only use redis as a celery result backend so the impact is fairly limited.